specification/operationalinsights/resource-manager/readme.md

```diff
@@ -38,6 +38,35 @@ openapi-type: arm
 tag: package-2023-09
 ```
 
+### Tag: package-2023-09
```

Comment:
>package-2023-09 [](http://example.com/codeflow?start=9&length=15)

It does not match with the new version added.

Reply:
Fixed.